docs/references/javascript/organization/members.mdx

```diff
@@ -10,41 +10,80 @@ These are all methods on [the Organization class](/docs/references/javascript/or
 ## `getMemberships()`
 
 ```typescript
-function getMemberships(params?: GetMembershipsParams): Promise<OrganizationMembership[]>;
+// Updated API
+function getMemberships(params?: GetMembershipsParams): Promise<PaginatedResponse<OrganizationMembership>>;
+
+// Deprecated API
+function getMemberships(params?: DeprecatedGetMembershipsParams): Promise<OrganizationMembership[]>;
 ```
 
 Retrieve the members of the selected
 
 ### `GetMembershipsParams`
 
+| Name | Type | Description |
+| --- | --- | --- |
+| `initialPage?` | number | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. |
+| `pageSize?` | number | A number that indicates the maximum number of results that should be returned for a specific page. |
+| `role?` | `('admin' \| 'basic_member' \| 'guest_member')[]` | The roles of memberships that will be included in the response |
+
+
+### Returns
+
+<Tables
+    headings={["Type", "Description"]}
+    headingsMeta={[{}, {maxWidth: "300px"}]}
+    rows={[
+        {
+            cells: [
+                <code>Promise{"<"}<a
+                    href="/docs/references/javascript/types/paginated-response">PaginatedResponse</a>{"<"}<a
+                    href="/docs/references/javascript/organization-domain">OrganizationMembership</a>{">"}</code>,
+                <>This method returns a <code>Promise</code> which resolves with a list of <code><a
+                    href="/docs/references/javascript/organization-domain">OrganizationMembership</a></code> objects.</>,
+            ]
+        },
+    ]}
+/>
+
+
+### `DeprecatedGetMembershipsParams`
 <Tables
-  headings={["Name", "Type", "Description"]}
-  headingsMeta={[{maxWidth:'300px'},{maxWidth: '300px'},{maxWidth: '300px'}]}
-  rows={[
-  {cells: [
-  <code>limit</code>,
-  <code>number | undefined</code>,
-  "Limit of the results returned.",
-  ]},
-  {cells: [
-  <code>offset</code>,
-  <code>number | undefined</code>,
-  "The offset of results to start the retrieval.",
-  ]},
-  ]}
+    headings={["Name", "Type", "Description"]}
+    headingsMeta={[{maxWidth: '300px'}, {maxWidth: '300px'}, {maxWidth: '300px'}]}
+    rows={[
+        {
+            cells: [
+                <code>limit</code>,
+                <code>number | undefined</code>,
+                "Limit of the results returned.",
+            ]
+        },
+        {
+            cells: [
+                <code>offset</code>,
+                <code>number | undefined</code>,
+                "The offset of results to start the retrieval.",
+            ]
+        },
+    ]}
 />
```

Comment:
Let's remove this table in favour of the `<Callout />` I added in the suggestion above ^